.gitignore

```diff
@@ -2,6 +2,8 @@
 /release
 /sht3x/sht3x_example_usage_hw_i2c
 /sht3x/sht3x_example_usage_sw_i2c
+/sht3x/user_config.inc
 /shtc1/shtc1_example_usage_hw_i2c
 /shtc1/shtc1_example_usage_sw_i2c
+/shtc1/user_config.inc
```

Comment:
We ignore these files, but they get added by this commit anyways. Is this intended?

Reply:
yes, we want the file checked in as template, but typically ignore changes the user makes. We'll have to `git add -f` to update the template